core/src/main/java/com/linecorp/armeria/common/HttpData.java

```diff
@@ -61,6 +61,13 @@ static HttpData empty() {
         return ByteArrayHttpData.EMPTY;
     }
 
+    /**
+     * Returns an empty {@link HttpData} whose HTTP/2 {@code endOfStream} flag is set with the specified value.
+     */
+    static HttpData empty(boolean endOfStream) {
+        return endOfStream ? ByteArrayHttpData.EMPTY_EOS : ByteArrayHttpData.EMPTY;
+    }
```

Comment:
Maybe no need to introduce this? i.e. `empty().withEndOfStream()` would work.

Reply:
If I didn't add this method, I thought somebody would propose this API.
Now I can completely remove this. 🤣